ImageGallery/src/org/sleuthkit/autopsy/imagegallery/gui/drawableviews/GroupPane.java

```diff
@@ -864,7 +864,7 @@ private ContextMenu buildContextMenu() {
 
             extractMenuItem.setOnAction(actionEvent -> {
                 SwingUtilities.invokeLater(() -> {
-                    TopComponent etc = WindowManager.getDefault().findTopComponent(ImageGalleryTopComponent.PREFERRED_ID);
```

Comment:
WindowManager now unused import.

Reply:
I removed the unused import and corrected the copyright.